redis/src/redis/endpoint.bal

```diff
@@ -23,7 +23,7 @@ public type Client client object {
     handle datasource;
 
     # Gets called when the endpoint is being initialized during the module initialization.
-    public function __init(ClientEndpointConfiguration config) returns Error? {
+    public function init(ClientEndpointConfiguration config) returns error? {
```

Comment:
We are using a distinct error in most of the connectors. I think, If we haven't any specific reason to change the `Error` to `error`, we can keep this.

Reply:
Reverted back in f8eade9